modules/core/include/opencv2/core/ocl.hpp

```diff
@@ -540,7 +540,7 @@ class CV_EXPORTS Kernel
     int64 runProfiling(int dims, size_t globalsize[], size_t localsize[], const Queue& q=Queue());
 
     size_t workGroupSize() const;
-    size_t preferedWorkGroupSizeMultiple() const;
```

Comment:
It's API change. I understand that there is a typo, but we do not want to break backward compatibility and existing code.

Reply:
I just reverted this one too